app/(app)/speakers.tsx

```diff
@@ -29,7 +29,18 @@ const getSpeakerSessions = () => {
 const SpeakersPage = () => {
   const { colors } = useTheme();
   const router = useRouter();
-  const data = getSpeakerSessions();
+
+  const { isLoading: isLoadingSpeakers, data: speakers } = useQuery({
+    queryKey: ['speakers'],
```

Comment:
We are using the same tanstack query key here to fetch data with a count of 50 while in the `services/api/index.ts` we are using the same query key to fetch data with a count of 15. This will result in messed up revalidation of this query since the count is definitely a dependency of the query and should be part of the query key.
(also check all the other queries for this problem)